collectors/snmpproxy.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+
+import os
+import sys
+import time
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                                'snmp'))
+from snmpraw import SNMPRawCollector
+from diamond.metric import Metric
+
+
+class SNMPProxyCollector(SNMPRawCollector):
+
+    def collect_snmp(self, device, host, port, community):
+        """
+        Collect SNMP interface data from device
+        """
+        self.log.debug(
+            'Collecting raw SNMP statistics from device \'{0}\''.format(device)
+        )
+
+        dev_config = self.config['devices'][device]
+        if 'oids' in dev_config:
+            for oid, metricName in dev_config['oids'].items():
+
+                if (device, oid) in self.skip_list:
+                    self.log.debug(
+                        'Skipping OID \'{0}\' ({1}) on device \'{2}\''.format(
+                            oid, metricName, device))
+                    continue
+
+                timestamp = time.time()
+                value = self._get_value(device, oid, host, port, community)
+                if value is None:
+                    continue
+
+                self.log.debug(
+                    '\'{0}\' ({1}) on device \'{2}\' - value=[{3}]'.format(
+                        oid, metricName, device, value))
+
+                device_path = '{}.{}.{}'.format(
+                    dev_config['instance_id'],
```

Comment:
why does instance id appear twice?
shouldn't it be `node_id.device.node_instance_id`?